dev/tasks/python-wheels/azure.linux.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         docker-compose -v
         cd arrow
         BUILD_IMAGE=centos-python-{{ wheel_tag }}
-        docker-compose pull $BUILD_IMAGE
+        docker-compose pull $BUILD_IMAGE || true
```

Comment:
We can use the `--ignore-pull-failures` option here: `docker-compose pull --ignore-pull-failures $BUILD_IMAGE`